quantecon/linprog_simplex.py

```diff
@@ -0,0 +1,327 @@
+"""
+Implements a linear programming solver using the "Simplex" method.
+
+"""
+
+import numpy as np
+from numba import jit
+from .util import pivot_operation, min_ratio_test, lex_min_ratio_test
+
+
+infeasible_err_msg = "The problem appears to be infeasible"
+unbounded_obj = "The problem appears to be unbounded."
+max_iter_p1 = "The maximum number of iterations has been reached in Phase I"
```

Comment:
Small thing, but in `max_iter_p2` you use "Phase 2" and in `max_iter_p1` you use "Phase I" -- Either change this to "Phase 1" or the other to "Phase II" just to be consistent